react/features/calendar-sync/middleware.js

```diff
@@ -178,10 +178,9 @@ function _fetchCalendarEntries(
  */
 function _getURLFromEvent(event, knownDomains) {
     const linkTerminatorPattern = '[^\\s<>$]';
-    /* eslint-disable max-len */
     const urlRegExp
-        = new RegExp(`http(s)?://(${knownDomains.join('|')})/${linkTerminatorPattern}+`, 'gi');
-    /* eslint-enable max-len */
+        = new RegExp(`http(s)?://(
+            ${knownDomains.join('|')})/${linkTerminatorPattern}+`, 'gi');
```

Comment:
Again, break inside `${}`: either right after `${` or right before `}`.

Reply:
Additionally, such formatting requires unnecessary attention on the side of the reader to parse. Break like I described in another comment:
```javascript
    = new RegExp(
        `http(s)?://(${knownDomains.join('|')})/${linkTerminatorPattern}+`,
        'gi');
```

If "        `http(s)?://(${knownDomains.join('|')})/${linkTerminatorPattern}+`" is more than 80 chars, then break further, for example:
```javascript
    = new RegExp(
        `http(s)?://(${knownDomains.join('|')})/${
            linkTerminatorPattern}+`,
        'gi');
```

or
```javascript
    = new RegExp(
        `http(s)?://(${knownDomains.join('|')
            })/${linkTerminatorPattern}+`,
        'gi');
```